RctGcs/droneSimulator.py

```diff
@@ -58,7 +58,7 @@
 import RCTComms.transport
 import utm
```

Comment:
Imports not sorted

Reply:
This should be here per https://peps.python.org/pep-0008/#imports